### PR TITLE
Updated the Data structures 

### PR DESCRIPTION
**Pull request by smritiprasad**
I have updated the descriptions of fields in Rate rules response, Book request, Cancel Request - Per Room and Cancel - Per Booking use case.

### CODE REVIEW

File: apis/universal-supplier-gateway/hotel/v1.0/_dataStructure/data-structures.apib [hunk 1]
```diff
@@ -1068,7 +1072,7 @@ There are two types of rates:<br><br>- `perBookingRate`<br>- `perRoomRate`<br><b
 + end: `2017-09-17T23:59:00` - End date and time for the window.<br><br>Format: [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)
 
 ## supplierValuePenalties
-+ supplierValue: 100 (number) - <a name="suppliervalue"></a>  Applicable penalty amount for the cancellation if the cancellation is made within the given cancellation window. This amount is applicable only when the `supplierValueType` field returned `Amount`. <br><br> Format: Decimal
++ supplierValue: 100 (number) - Applicable `penalty value` for the cancellation if the cancellation is made within the given cancellation window. If the `supplierValueType` returns `amount`, then the applicable `supplierValue` is amount charged as penalty. If the `supplierValueType` returns `nights`, then the applicable `supplierValue` is number of nights charged as penalty. Likewise, if the `supplierValueType` returns `percent`, then the applicable `supplierValue` is percent to be charged as penalty.<br><br> Format: Decimal
```
Comment (mirairani) [inline apis/universal-supplier-gateway/hotel/v1.0/_dataStructure/data-structures.apib]: delete all conditions from this parameter and add it in supplierValueTypePenalties per the edits provided for the conditions.

Comment (mirairani) [inline apis/universal-supplier-gateway/hotel/v1.0/_dataStructure/data-structures.apib]: Add the following line:
The penalty value can either be a specific amount, a percentage of the total room cost, or a specific number of room nights. For more information, see `supplierValueType`.

Comment (mirairani) [inline apis/universal-supplier-gateway/hotel/v1.0/_dataStructure/data-structures.apib]: `penalty value` should be penalty value

**Comment by smritiprasad** [comment]
Updated the changes per your review comments. Please review.

**Comment by smritiprasad** [comment]
Removed the quotes from penalty value mentioned in field + supplierValue

**Pull request merged**